fractal_server/migrations/versions/eebd9f68ad58_add_dumps_to_applyworkflow.py

```diff
@@ -0,0 +1,98 @@
+"""Add dumps to ApplyWorkflow
+
+Revision ID: eebd9f68ad58
+Revises: 99ea79d9e5d2
+Create Date: 2023-10-25 12:46:29.503628
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "eebd9f68ad58"
+down_revision = "99ea79d9e5d2"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("applyworkflow", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "user_dump", sa.JSON(), server_default="{}", nullable=False
+            )
+        )
+        batch_op.add_column(
+            sa.Column(
+                "input_dataset_dump",
+                sa.JSON(),
+                server_default="{}",
+                nullable=False,
+            )
+        )
+        batch_op.add_column(
+            sa.Column(
+                "output_dataset_dump",
+                sa.JSON(),
+                server_default="{}",
+                nullable=False,
+            )
+        )
+        batch_op.alter_column(
+            "workflow_dump",
+            existing_type=postgresql.JSON(astext_type=sa.Text()),
```

Comment:
I'm confused by this kind of `alter_column` arguments. Are they autogenerated by alembic? Why do they differ from what is above? And most importantly: why is there a reference to postgresql at all?